LICENSE-BSD

```diff
@@ -0,0 +1,11 @@
+Copyright 2024 ANTLR Kotlin contributors
```

Comment:
The license is provided at the repo level, see https://github.com/antlr/antlr5/blob/dev/LICENSE.txt so this file should be removed